src/components/Publishing/PublishingProposalsList/PublishingProposalsListItem.vue

```diff
@@ -23,6 +23,14 @@
               Submitted on
               <strong>{{ updated }}</strong>
             </p>
+            <template>
+              <div v-if="email">
+              <p class="dataset-meta">
+                Submitter's email address
```

Comment:
This label may be too long. How does it look without the word "Submitter's"?

Reply:
I can change to just 'Email address'. Looks better